src/gsSolver/gsMatrixOp.h

```diff
@@ -126,9 +143,9 @@ class gsMatrixOp : public gsLinearOperator<typename MatrixType::Scalar>
   * \ingroup Solver
   */
 template <class Derived>
-typename gsMatrixOp<Derived>::uPtr makeMatrixOp(const Eigen::EigenBase<Derived>& mat, bool sym=false)
```

Comment:
"sym" flags make things complicated and I have received a lot of complaints for them.
Does it really make something a lot faster ? i.e., is it really needed ?

Reply:
That's why I _remove_ those flags.